organizing-meetups.md

```diff
@@ -50,3 +50,25 @@ Freelancer
 ProudCloud
 
 SourcePad
+
+####  Filta
+PhRUG contacts: @padi
+
+##### Address
```

Comment:
Just for consistency - maybe just make this a normal `p`? Or make the Angat one an h4 as well?

Reply:
Done!